pages/android/wm.md

```diff
@@ -0,0 +1,12 @@
+# wm
+
+> Show information about the screen of an Android device.
+> More information: <https://adbinstaller.com/commands>.
```

Comment:
Is there a more specific URL here? That one doesn't seem very specific.

Reply:
@sbrl https://github.com/tldr-pages/tldr/pull/5457#discussion_r595099178

Reply:
Hrm. While the hash is somewhat concerning, I'm not convinced of the usefulness of the current link with respect to the specific command at hand.

It doesn't even look like an official source (though I may be wrong on that).

Reply:
I couldn't find any official documentation. It seems that android commands are poorly documentated in general.

Reply:
I see. In that case, I think we should leave the more information link out until a good official source of documentation can be linked to. No link at all is better than sending users to bad documentation.